### PR TITLE
Update configure-pod-configmap.md

### PR DESCRIPTION
**Pull request by purebluesong**

### CODE REVIEW

File: content/zh/docs/tasks/configure-pod-container/configure-pod-configmap.md [hunk 1]
```diff
@@ -994,8 +994,8 @@ annotations.
 `kubelet` 在每次定期同步时都会检查已挂载的 ConfigMap 是否是最新的。
 但是，它使用其本地的基于 TTL 的缓存来获取 ConfigMap 的当前值。
 因此，从更新 ConfigMap 到将新键映射到 Pod 的总延迟可能与
-kubelet 同步周期 + ConfigMap 在 kubelet 中缓存的 TTL 一样长。
-
+kubelet 同步周期（默认1分钟） + ConfigMap 在 kubelet 中缓存的 TTL（默认1分钟） 一样长。
+可以通过更新 Pod 的 Annotation 触发立即更新。
```
Comment (tengqm) [inline content/zh/docs/tasks/configure-pod-container/configure-pod-configmap.md:998]:
```suggestion
可以通过更新 Pod 的注解触发立即更新。
```

**Review by Arhell — COMMENTED** [review]
Pls sign the CLA
more info https://github.com/kubernetes/website/pull/29824#issuecomment-927507807

**Comment by jimangel** [comment]
Hey there! @purebluesong, looks like you haven't signed the CLA yet. Could I please have you do that? https://github.com/kubernetes/community/blob/master/CLA.md

**Comment by purebluesong** [comment]
> Hey there! @purebluesong, looks like you haven't signed the CLA yet. Could I please have you do that? https://github.com/kubernetes/community/blob/master/CLA.md

Sorry about that. Something happens when I try to sign CLA. It shows "Access denied
You are not authorized to access this page." in https://identity.linuxfoundation.org/projects/cncf/individual-signup.

After logged linuxfoundation with another account, I found that I can't bind my github account with the linux foundation account and I could't remove the binding relation between old account and github.

I have send some issues to linuxfounddation but not receive any feedback. I have no idea about what should I do now.

**Comment by tengqm** [comment]
CLA is still failing. No response from the contributor.
/close

**Pull request closed without merging**